plaidml/op/lib/ops.cc

```diff
@@ -431,7 +431,7 @@ std::vector<int64_t>* extend_manual_padding(std::vector<int64_t>* pads, size_t r
 }  // namespace
 
 Value abs(const Value& value) {
-  IVLOG(1, "abs");
+  IVLOG(2, "abs");
```

Comment:
Hmm, I think these are good to have at this level.

Reply:
Why specifically?  I'd like to keep log level 1 to basic status, basically something with O(1) lines per program execution or thereabouts.  Basically if you run resnet50 at log level 1, it spews > 2k lines prior to this commit.

Reply:
Yeah, this should be happening at the O(1) level.

Reply:
What's the verdict?  O(1) or O(2)?  These logs aren't very illuminating, in my opinion.  How does logging the operation name help?

Reply:
Usually, this comes up like something crashed and it'd be interesting to see which operation caused it. The `O(1)` means that we expect the log to be printed once (not like `O(n)` or `O(n*n)`). Overall it's not a big deal, I just didn't think the oplib logs were excessive so far.

Reply:
OK, I think it is reasonable to put these back.  Slight increase in log size for ResNet50.  235535 vs. 196395.  Same order of magnitude.